graylog2-web-interface/src/components/authentication/BackendWizard/ConnectionErrors.jsx

```diff
@@ -0,0 +1,48 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import styled, { type StyledComponent } from 'styled-components';
+
+import type { ThemeInterface } from 'theme';
+import { Alert } from 'components/graylog';
+
+export const NotificationContainer: StyledComponent<{}, ThemeInterface, Alert> = styled(Alert)`
+  margin-top: 10px;
+`;
+
+const ErrorsTitle = styled.div`
+  font-weight: bold;
+  margin-bottom: 5px;
+`;
+
+const ErrorsList = styled.ul`
+  list-style: initial;
+  padding-left: 20px;
+`;
+
+type Props = {
+  errors: Array<string>,
+  message: string,
+};
+
+const ConnectionErrors = ({ errors, message }: Props) => (
+  <NotificationContainer bsStyle="danger">
+    <ErrorsTitle>{message}</ErrorsTitle>
+    <ErrorsList>
+      {errors.map((error) => {
+        return <li key={error}>{error}</li>;
```

Comment:
I am not against using a `mono` font, but in all other error cases I found we are using the default font.

Reply:
As it might be, here we are displaying `raw` data. It is not created in the frontend but in the backend. We might not even know how it looks, because it might be different depending on our authentication server we try to test again:

So I prefer:
![Graylog - Edit LDAP Authentication Service](https://user-images.githubusercontent.com/448763/95215556-62247000-07f1-11eb-992a-efd02014c905.png)

over this:
![Graylog - Edit LDAP Authentication Service (1)](https://user-images.githubusercontent.com/448763/95215583-694b7e00-07f1-11eb-81c3-e3e17d634841.png)